docs/semgrep-appsec-platform/dashboard-old.md

```diff
@@ -0,0 +1,154 @@
+---
```

Comment:
If I try to go to this doc, it redirects to the current dashboard doc, right (this is what I see, so I just want to confirm that's correct)? If so, should we delete this doc?

Reply:
Yeah, I temporarily named it dashboard-old for now, and we'll delete it after 1-2 months if it looks like we won't be reverting or keeping any information from this.